src/librustc/ich/impls_hir.rs

```diff
@@ -828,7 +828,7 @@ impl_stable_hash_for!(struct hir::EnumDef {
 });
 
 impl_stable_hash_for!(struct hir::VariantKind {
-    name,
+    ident,
```

Comment:
```suggestion
    ident -> (ident.name),
```

Reply:
Are you still getting those linking errors?
Instead of newly introduced idents try hashing their names (both here in `librustc/ich/impls_hir.rs` and in `librustc/ty/mod.rs`.)
This is also important to avoid regressing incremental compilation.

Reply:
This was previously discussed in https://github.com/rust-lang/rust/pull/51492.

Reply:
No, I solved them with this very change already. I guess you're looking at an old commit? I may not have pushed, sorry.